src/Zoomba/GUILibrary.py

```diff
@@ -436,3 +436,22 @@ def get_react_list_labels(self, locator):
         react_select_container = self.find_element(locator)
         options = RS.ReactSelect(react_select_container).options()
         return [opt.text for opt in options]
+
+    @keyword("Select From Search Field")
+    def select_from_search_field(self, locator, text, timeout=None):
+        """This is a Selenium keyword that first attempts to find a web element, clears it, and then inputs text.
+        Subsequently, it selects the first item in the search dropdown. 
+        If typing into the element fails, the keyword will scroll to the bottom of the page and try again.
```

Comment:
Does it really do this? I don't get any indication from the code that this is the case.